setup.py

```diff
@@ -323,6 +324,7 @@ def run(self):
     deps["sacremoses"],  # for XLM
     deps["tokenizers"],
     deps["tqdm"],  # progress bars in model download and training scripts
+    deps["detectron2"],  # detectron2 for image embedding
```

Comment:
use 
`deps["detectron2 @ git+https://github.com/facebookresearch/detectron2.git"], # detectron2 for image embedding`

as documented in https://stackoverflow.com/questions/60414753/how-to-install-githttps-from-setup-py-using-install-requires

Reply:
still giving error saying that torch is not available . any help please

Reply:
You need to change this at all points where detectron2 is used. Did you run the tests locally? Also, it's `detectron2 @ git+https://github.com/facebookresearch/detectron2.git` (with a space). I will tag where you have to replace it